cmd/gaurun/gaurun.go

```diff
@@ -107,7 +107,7 @@ func main() {
 	}
 	if gaurun.ConfGaurun.Ios.Enabled {
 		if err := gaurun.InitAPNSClient(); err != nil {
-			gaurun.LogSetupFatal(fmt.Errorf("failed to init http client"))
+			gaurun.LogSetupFatal(fmt.Errorf("failed to init http client: %v", err))
```

Comment:
Committed(https://github.com/mercari/gaurun/pull/63/commits/e2ad0b664de8f68cc6d8a53026c8773fc171b5bc). That's better.